cmake/modules/SwiftUtils.cmake

```diff
@@ -190,33 +190,29 @@ endfunction()
 # we create a `swift-driver` symlink adjacent to the `swift` and `swiftc` executables
 # to ensure that `swiftc` forwards to the standalone driver when invoked.
 function(swift_create_early_driver_copies target)
-  # Early swift-driver is built adjacent to the compiler (swift build dir)
-  if(CMAKE_BUILD_TYPE STREQUAL "Release" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
-    set(driver_package_configuration_dir "release")
-  else()
-    set(driver_package_configuration_dir "debug")
+  set(SWIFT_EARLY_SWIFT_DRIVER_BUILD "" CACHE PATH "Path to early swift-driver build")
+
+  if(NOT SWIFT_EARLY_SWIFT_DRIVER_BUILD)
+    return()
   endif()
-  
-  set(driver_bin_dir "${CMAKE_BINARY_DIR}/../earlyswiftdriver-${SWIFT_HOST_VARIANT}-${SWIFT_HOST_VARIANT_ARCH}/${driver_package_configuration_dir}/bin")
-  set(swift_bin_dir "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
-  # If early swift-driver wasn't built, nothing to do here.
-  if(NOT EXISTS "${driver_bin_dir}/swift-driver" OR NOT EXISTS "${driver_bin_dir}/swift-help")
-      message(STATUS "Skipping creating early SwiftDriver symlinks - no early SwiftDriver build found at: ${driver_bin_dir}.")
-      return()
+
+  if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+    set(_CMAKE_HOST_EXECUTABLE_SUFFIX .exe)
```

Comment:
Can we use `CMAKE_EXECUTABLE_SUFFIX`? It should match whatever we're building the frontend for.

Reply:
What if we are cross-compiling?

Reply:
Then nothing works anyway because we use this driver and the just-built swift-frontend later to build the stdlib.
At the very least, both the frontend and early swift driver should be built for the same thing.